src/docker/workflows/cp-schema-registry.ts

```diff
@@ -218,7 +222,7 @@ export class ConfluentPlatformSchemaRegistryWorkflow extends LocalResourceWorkfl
     }
 
     const kafkaImageRepo: string = getLocalKafkaImageName();
-    const kafkaImageTag: string = kafkaWorkflow.imageTag;
+    const kafkaImageTag: string = getLocalKafkaImageTag();
```

Comment:
Unrelated-to-this-PR bug fix where we were incorrectly using the `latest` (default) tag instead of whatever was configured in the user/workspace settings.